test/suite/test_txn24.py

```diff
@@ -52,6 +58,9 @@ def conn_config(self):
                 eviction=(threads_max=4)'
 
     def test_snapshot_isolation_and_eviction(self):
+        if sys.platform.startswith('darwin'):
+            # Skip this test on MacOS/Darwin as it causes WT_ROLLBACKs to occur frequently, and they generate errors.
```

Comment:
I think we should prepend `FIXME-WT-9575` in the comment.

Reply:
Done!